imports/include/niFakeNonIvi.h

```diff
@@ -1,8 +1,9 @@
 #ifndef NIFAKE_NON_IVI_H
 #define NIFAKE_NON_IVI_H
 #include <cstdint>
+// we need the typedefs from here
+#include <NIDAQmx.h>
```

Comment:
We talked about removing this and having fake non-ivi use stdint types instead. If you want, make a follow on task for this.

Reply:
Sorry, I thought we ended up at a different place on this. But this is good, and I made this change.